internal/updater/check.go

```diff
@@ -98,7 +98,7 @@ func AutoUpdate(pjPath string, out output.Outputer) (updated bool, resultVersion
 		if os.IsPermission(errs.InnerError(err)) {
 			out.Error(locale.T("auto_update_permission_err"))
 		}
-		logging.Error("Unable to self update: %s", err)
+		logging.Debug("Unable to self update: %s", err)
```

Comment:
Now all errors are disregarded. I was saying you still identify the cause of the error, and only if you cannot identify the cause do you log it as an error.